docs/modules/getting_started/examples/code/getting-started.sh

```diff
@@ -0,0 +1,119 @@
+  exit 1
+fi
+
+case "$1" in
+"helm")
+echo "Adding 'stackable-dev' Helm Chart repository"
+# tag::helm-add-repo[]
+helm repo add stackable-dev https://repo.stackable.tech/repository/helm-dev/
+# end::helm-add-repo[]
+
+echo "Installing Operators with Helm"
+# tag::helm-install-operators[]
+helm install --wait commons-operator stackable-dev/commons-operator --devel
+helm install --wait secret-operator stackable-dev/secret-operator --devel
+helm install --wait hive-operator stackable-dev/hive-operator --devel
+# end::helm-install-operators[]
+
+echo "Install minio certificates from minio-certificates.yaml"
+# tag::helm-install-minio-certificates[]
+kubectl apply -f minio-certificates.yaml
+# end::helm-install-minio-certificates[]
+
+echo "Install minio for S3"
+# tag::helm-install-minio[]
+helm install minio \
+--namespace default \
+--version 4.0.2 \
+--set mode=standalone \
+--set replicas=1 \
+--set persistence.enabled=false \
+--set buckets[0].name=hive,buckets[0].policy=none \
+--set users[0].accessKey=hive,users[0].secretKey=hivehive,users[0].policy=readwrite \
+--set resources.requests.memory=1Gi \
+--set service.type=NodePort,service.nodePort=null \
+--set consoleService.type=NodePort,consoleService.nodePort=null \
+--set tls.enabled=true,tls.certSecret=minio-tls-certificates,tls.publicCrt=tls.crt,tls.privateKey=tls.key \
+--repo https://charts.min.io/ minio
+# end::helm-install-minio[]
+
+echo "Install postgres for Hive"
+# tag::helm-install-postgres[]
+helm install postgresql \
+--version=10 \
+--namespace default \
+--set postgresqlUsername=hive \
+--set postgresqlPassword=hive \
+--set postgresqlDatabase=hive \
+--repo https://charts.bitnami.com/bitnami postgresql
+# end::helm-install-postgres[]
+;;
+"stackablectl")
+echo "installing Operators with stackablectl"
+# tag::stackablectl-install-operators[]
+stackablectl operator install commons secret hive
+# end::stackablectl-install-operators[]
+
+echo "installing MinIO and PostgreSQL with stackablectl"
+# tag::stackablectl-install-minio-postgres-stack[]
+stackablectl \
+--additional-stacks-file stackablectl-postgres-minio-stack.yaml \
+--additional-releases-file release.yaml \
+stack install minio-postgres
+# end::stackablectl-install-minio-postgres-stack[]
+;;
+*)
+echo "Need to provide 'helm' or 'stackablectl' as an argument for which installation method to use!"
+exit 1
+;;
+esac
+
+echo "Install HiveCluster"
+# tag::install-hive[]
+kubectl apply -f hive-minio-credentials.yaml
+kubectl apply -f hive-minio-credentials-secret-class.yaml
+kubectl apply -f hive-minio-s3-connection.yaml
+kubectl apply -f hive-postgres-s3.yaml
+# end::install-hive[]
+
+sleep 5
+
+echo "Awaiting Hive rollout finish"
+# tag::watch-hive-rollout[]
+kubectl rollout status --watch statefulset/hive-postgres-s3-metastore-default
+# end::watch-hive-rollout[]
+
+echo "Install Hive test helper from hive-test-helper.yaml"
+# tag::install-hive-test-helper[]
+kubectl apply -f hive-test-helper.yaml
```

Comment:
I think if you're not using this bit in the actual documentation it's not necessary to have this here, we already have it in the kuttl tests.

This script should just test what's in the docs.

Reply:
As discussed this is for testing the cluster afterwards and make sure the doc examples work together